jina/types/document/__init__.py

```diff
@@ -51,18 +49,11 @@
         scipy.sparse.bsr_matrix,
         scipy.sparse.csc_matrix,
         torch.sparse_coo_tensor,
-        tf.SparseTensor,
-    )
-
-    SparseArrayType = TypeVar(
-        'SparseArrayType',
-        np.ndarray,
-        scipy.sparse.csr_matrix,
-        scipy.sparse.coo_matrix,
-        scipy.sparse.bsr_matrix,
-        scipy.sparse.csc_matrix,
-        torch.sparse_coo_tensor,
-        tf.SparseTensor,
+        tensorflow.SparseTensor,
+        tensorflow.Tensor,
+        torch.Tensor,
```

Comment:
torch Tensor is never returned by our embeddings or blobs

Reply:
yes, just to be comprehensive for future